ramls/schemas/fund_distribution.json

```diff
@@ -31,8 +31,5 @@
     }
   },
   "additionalProperties": false,
-  "required": [
-    "name",
-    "code"
-  ]
+  "required": null
```

Comment:
IIUC then the `required` property can be completely removed if needed to be empty.